### PR TITLE
Fix spelling and typos found by codespell

### PR DESCRIPTION
**Pull request by leeks-int**
Signed-off-by: Alexey Oralov alexey.oralov@intel.com

### CODE REVIEW

File: jenkinsfiles/UB20_Check.groovy [hunk 1]
```diff
@@ -80,7 +80,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```
Comment (ValentinaKats) [inline jenkinsfiles/UB20_Check.groovy:83]: @DoyleLi, are you ok with such variable renaming? I have found you're the author of this file.

File: jenkinsfiles/UB20_Check.groovy [hunk 2]
```diff
@@ -80,7 +80,7 @@ pipeline {
...
         def DATESTEMP = sh(script: "date +\"%Y-%m-%d\"", returnStdout: true).trim()
```
Comment (timmiesmith) [inline jenkinsfiles/UB20_Check.groovy:84]: DATESTEMP should be changed to DATESTAMP to be consistent if the timestamp change is made.

File: jenkinsfiles/RHEL_Check.groovy [hunk 3]
```diff
@@ -95,7 +95,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```
Comment (timmiesmith) [inline jenkinsfiles/RHEL_Check.groovy:98]: TIMESTAMP is the correct spelling, and DATESTEMP should be changed to DATESTAMP to be consistent. @DoyleLi, is this change ok in all of the Jenkins files?

File: jenkinsfiles/UB18_Check.groovy [hunk 4]
```diff
@@ -95,7 +95,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```
Comment (timmiesmith) [inline jenkinsfiles/UB18_Check.groovy:98]: TIMESTAMP is correct, but DATESTEMP should also be changed to DATESTAMP to be consistent.

File: jenkinsfiles/Windows_Check.groovy [hunk 5]
```diff
@@ -105,7 +105,7 @@ pipeline {
 
     environment {
         def NUMBER = sh(script: "expr ${env.BUILD_NUMBER}", returnStdout: true).trim()
-        def TIMESTEMP = sh(script: "date +%s", returnStdout: true).trim()
+        def TIMESTAMP = sh(script: "date +%s", returnStdout: true).trim()
```
Comment (timmiesmith) [inline jenkinsfiles/Windows_Check.groovy:108]: TIMESTAMP is correct, but DATESTEMP should be changed to DATESTAMP to be consistent.

**Review by timmiesmith — CHANGES_REQUESTED** [review]

File: jenkinsfiles/UB20_Check.groovy (hunk 1 above)
Comment (DoyleLi) [reply jenkinsfiles/UB20_Check.groovy:83]: Hi, it looks good to me. Thanks for notifying.

Comment (leeks-int) [reply jenkinsfiles/UB20_Check.groovy:83]: Updated

**Review by timmiesmith — APPROVED** [review]

File: jenkinsfiles/RHEL_Check.groovy (hunk 3 above)
Comment (DoyleLi) [reply jenkinsfiles/RHEL_Check.groovy:98]: Hi, Yes, it is OK to change the spelling everywhere.

**Review by ValentinaKats — APPROVED** [review]

**Pull request merged**